external-secrets-operator.yaml

```
@@ -26,7 +26,7 @@ pipeline:
   - runs: |
       make build-$(go env GOARCH)
       mkdir -p ${{targets.destdir}}/usr/bin
-      mv bin/external-secrets* ${{targets.destdir}}/usr/bin/
+      mv bin/external-secrets* ${{targets.destdir}}/usr/bin/external-secrets
```

```suggestion
      mv bin/external-secrets ${{targets.destdir}}/usr/bin/external-secrets
```

Or if there are multiple bins then /usr/bin/